### PR TITLE
2.a.2 - Added more data (Increased total ODA figures)

### PR DESCRIPTION
**Pull request by MaxEngledew**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

**Comment by MaxEngledew** [comment]
Seems to be sighting the lack of year in the first column as the issue - although,  I can see "Year" in the data editor.

File: data/indicator_2-a-2.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Year,Type of ODA,Aid Broad Category Description (DAC 5 Code),Aid Description (CRS Code),Value
+"Year ",Type of ODA,Aid Broad Category Description (DAC 5 Code),Aid Description (CRS Code),Value
```
Comment (shaneporter) [inline data/indicator_2-a-2.csv]: this line now has quotes around the Year field, plus a space.

Comment (dougmet) [inline data/indicator_2-a-2.csv]: quotes don't matter. It's the spaces

Comment (MaxEngledew) [inline data/indicator_2-a-2.csv]: So submit new pull request, ensuring to remove the space?

Comment (dougmet) [inline data/indicator_2-a-2.csv]: I'll fix it in this PR

**Comment by dougmet** [comment]
Fixed and merged.

**Pull request merged**